Project/Source/Modules/ModulePhysics.h

```diff
@@ -90,9 +89,9 @@ class ModulePhysics : public Module {
 	btSequentialImpulseConstraintSolver* constraintSolver = nullptr;
 	btDiscreteDynamicsWorld* world = nullptr;
 
-	/* BULLET DEBUG: Uncomment to activate it
-	DebugDrawer* debugDrawer;
-	*/
+	//BULLET DEBUG: Uncomment to activate it
+	//DebugDrawer* debugDrawer;
+
 	bool debug = true;
 };
```

Comment:
I was looking at it after we talked about it, this debug variable is only used in the Update() debugging code, so it can be commented too. Actually the commented if condition in Update() might be completely unnecessary.

Reply:
If you only one execute code for debugging purposes you can use the prepocessoar macro #if __DEBUG

Reply:
But we don't even want when we have the engine in debug normally. So it would be commented anyway inside the macro.